scanpy/tests/test_preprocessing.py

```diff
@@ -91,18 +91,37 @@ def test_normalize_per_cell():
         axis=1).A1.tolist()
 
 
-def test_subsample():
+def test_sample():
```

Comment:
Could there be tests that the values with the object are correct, not just the shape?

I'm thinking something like using an identity matrix as `X` and making sure each row or column on the sampled axis still has only one value.

Reply:
If you feel like contributing this, feel free, otherwise I’ll just rely on numpy working as expected.